datafusion/sqllogictest/test_files/order.slt

```diff
@@ -769,18 +769,18 @@ SortPreservingMergeExec: [m@0 ASC NULLS LAST,t@1 ASC NULLS LAST]
 --SortExec: expr=[m@0 ASC NULLS LAST,t@1 ASC NULLS LAST]
 ----InterleaveExec
 ------ProjectionExec: expr=[Int64(0)@0 as m, t@1 as t]
---------AggregateExec: mode=FinalPartitioned, gby=[Int64(0)@0 as Int64(0), t@1 as t], aggr=[]
+--------AggregateExec: mode=FinalPartitioned, gby=[Int64(0)@0 as Int64(0), t@1 as t], aggr=[], ordering_mode=PartiallySorted([0])
```

Comment:
In some test, mode of the aggregateExec changes. Because now we can determine they contains ordered expressions (e.g constants are treated as if they are ordered).